outport/process/alteredaccounts/alteredAccountsProvider_test.go

```diff
@@ -530,13 +535,19 @@ func testExtractAlteredAccountsFromPoolShouldNotIncludeReceiverAddressIfNftCreat
 	aap, _ := NewAlteredAccountsProvider(args)
 
 	res, err := aap.ExtractAlteredAccountsFromPool(&outportcore.Pool{
+		Txs: map[string]data.TransactionHandlerWithGasUsedAndFee{
+			"hh": outportcore.NewTransactionHandlerWithGasAndFee(&transaction.Transaction{
+				SndAddr: []byte("sender in shard 0 - tx 1  "),
```

Comment:
Please extract `sender in shard 0 - tx 1` as well as its hex encoding(senderInHex = hex.EncodeTostring(...) = `73656e64657220696e2073686172642030202d20747820312020`) in separate variables and use them below.
Otherwise, it's hard to follow that the hex encoding of the above string is correct or not

Reply:
done